db/languages.yml

```diff
@@ -7,24 +7,30 @@ c++:
   :lexer: c++
   :compiled: yes
   :interpreted: no
-  :current: c++11
+  :current: c++17
   :extension: .cpp
   :source_filename: program
   :exe_extension: .exe
   :processes: 1
   :variants:
-    c++03:
+    c++03:  # depreciated
       :name: "C++03"
       :compiler: /usr/bin/g++
-      :compiler_command: "%{compiler} -std=gnu++03 -O2 -o %{output} %{source} -lm"
-#      upgrade: c++11 # submissions will be upgraded if possible, if support is removed, it will be force-upgraded
-#      note: probably will never happen for this particular example... (or at least not for the next 10 years)
+      :compiler_command: "%{compiler} --version 1>&2 && %{compiler} -std=gnu++03 -O2 -o %{output} %{source} -lm"
```

Comment:
How about we only show the first line of the version (without the copyright)?
```
"%{compiler} --version | head -n 1 >&2 && ..."
```

Reply:
Good idea